webapp/channels/src/components/integrations/abstract_command.tsx

```diff
@@ -385,15 +379,15 @@ export default class AbstractCommand extends React.PureComponent {
                                 <input
                                     id='description'
                                     type='text'
-                                    maxLength='128'
+                                    maxLength={128}
                                     className='form-control'
                                     value={this.state.description}
                                     onChange={this.updateDescription}
                                 />
                                 <div className='form__help'>
                                     <FormattedMessage
                                         id='add_command.description.help'
-                                        defaultMessage='Describe your slash command.'
+                                        defaultMessage='Describe your incoming webhook.'
```

Comment:
Why do we need to change the default message from slash command to incoming webhook?